### PR TITLE
ycsb: pull various improvements from kv

### PR DESCRIPTION
**Pull request by petermattis**
Allow specifying multiple databases to talk to on the command line.

Copy the kv splitting code which splits and scatters in parallel.

### CODE REVIEW

**Comment by petermattis** [comment]
It might be time soon to investigate sharing code between `ycsb`, `kv` and other load generators. But not yet.

File: ycsb/main.go [hunk 1]
```diff
@@ -501,8 +554,10 @@ func (m *mongo) clone() database {
 	}
 }
 
-func setupMongo(parsedURL *url.URL) (database, error) {
-	session, err := mgo.Dial(parsedURL.String())
+func setupMongo(dbURLs []string) (database, error) {
+	// NB: Mongo automatically detects the other nodes in the cluster. We just
```
Comment (rjnn) [inline ycsb/main.go]: s/Mongo/the Mongo driver/

**Review by rjnn — APPROVED** [review]
LGTM

File: ycsb/main.go (hunk 1 above)
Comment (petermattis) [reply ycsb/main.go]: Done.

**Pull request merged**